internal/pkg/utils/error.go

```diff
@@ -11,50 +12,66 @@ import (
 
 type multiError = multierror.Error
 
-type Error struct {
+type MultiError struct {
 	*multiError
 	lock *sync.Mutex
 }
 
-type RawError struct {
```

Comment:
`RawError` sa pouzival na vypisovanie nejakych informacii vramci chyby, pouzivalo sa to malo, ... a je to nahradene s `logger.Info()`.